cmd/protoc-gen-gorums/gengorums/gorums_bundle.go

```diff
@@ -27,13 +28,13 @@ const (
 
 // GenerateBundleFile generates a file with static definitions for Gorums.
 func GenerateBundleFile(dst string) {
-	pkgIdent, code := bundle(devPkgPath)
+	pkgIdent, reservedIndents, code := bundle(devPkgPath)
```

Comment:
should it be `reservedIdents` instead of `reservedIndents`?

Reply:
fixed